TheWarWithin/Priorities/DruidBalance.simc

```diff
@@ -38,9 +38,9 @@ actions+=/run_action_list,name=st
 actions.aoe+=/wrath,if=variable.enter_lunar&variable.eclipse&variable.eclipse_remains<cast_time
 actions.aoe+=/starfire,if=!variable.enter_lunar&variable.eclipse&variable.eclipse_remains<cast_time
 actions.aoe+=/starfall,if=astral_power.deficit<=variable.passive_asp+6
-actions.aoe+=/moonfire,cycle_targets=1,if=refreshable&(target.time_to_die-remains)>6&astral_power.deficit>variable.passive_asp+energize_amount&!boss
+actions.aoe+=/moonfire,cycle_targets=1,if=(!talent.treants_of_the_moon|spell_targets-active_dot.moonfire_dmg>6|cooldown.force_of_nature.remains>3&!buff.harmony_of_the_grove.up)
```

Comment:
I don't think we have `moonfire_dmg` implemented; can change these to `moonfire` or revise the aura definition:

```Lua
    -- Suffering $w2 Arcane damage every $t2 seconds.$?$w8<0[; Movement slowed by $w8%.][]
    moonfire = {
        id = 164812,
        duration = function () return mod_circle_dot( level > 13 and 22 or 18 ) end,
        tick_time = function () return mod_circle_dot( 2 ) * ( 1 - 0.125 * talent.cosmic_rapidity.rank ) * haste end,
        type = "Magic",
        max_stack = 1,
+        copy = { 155625, "moonfire_dmg" }
    },
```

Reply:
Better to go with the copy so we don't need to translate APL every time